src/database/models/repository.py

```diff
@@ -26,4 +28,5 @@ class Repository(base_model()):
     repo_type = Column(Integer)
     nickname = Column(String(255))      # Allow for a custom nickname to be set on the repo.
     should_display = Column(Boolean)    # Allow the repo to be hidden on the site
-    is_private = Column(Boolean)    # Flag for whether or not the repo is private
\ No newline at end of file
+    is_private = Column(Boolean)    # Flag for whether or not the repo is private
+    languages = relationship("Language", secondary=repo_language_table, back_populates='repositories')
```

Comment:
Might be useful for force 'joined' eager style here.
http://docs.sqlalchemy.org/en/latest/orm/loading_relationships.html#joined-eager-loading
http://docs.sqlalchemy.org/en/latest/orm/loading_relationships.html#configuring-loader-strategies-at-mapping-time

Because you know for a fact that you'll always need the languages along with the repo (unless that changes later) it makes since to always join this data in one select statement than do one query to get the repo and another to join.